### PR TITLE
Norwegian Bokmål translation

### PR DESCRIPTION
**Pull request by comradekingu**
I don't want this in Crowdin, and I am hoping you can be convinced to use Weblate instead :)
Edit: I can help set it up, and will fix the source strings if it is done.

### CODE REVIEW

File: app/src/main/res/values-nb-rNO/strings.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <string name="add">Legg til</string>
+    <string name="invalid_token">Angitt symbol er ugyldig.</string>
+    <string name="delete">Slett</string>
+    <string name="edit">Rediger</string>
+    <string name="restore_defaults">Gjenopprett forvalg</string>
+    <string name="no_keys">Ingen OTP-nøkler er installert.</string>
+    <string name="add_token">Legg til symbol</string>
+    <string name="scan_qr_code">Skann QR-kode</string>
+    <string name="interval">Intervall</string>
+    <string name="counter">Counter</string>
```
Comment (comradekingu) [inline app/src/main/res/values-nb-rNO/strings.xml:12]: I couldn't figure out what this is.

Comment (helloworld1) [reply app/src/main/res/values-nb-rNO/strings.xml:12]: This is used for hotp. Try add a token and select hotp. You will see counter

**Comment by helloworld1** [comment]
> I don't want this in Crowdin, and I am hoping you can be convinced to use Weblate instead :)
> Edit: I can help set it up, and will fix the source strings if it is done.


You don't necessary need to use Crowdin. I actually only care about the pull requests. It's up to the translator to decide the tool to use.

**Comment by 6543** [comment]
well :point_up:  - you dont have to use it ...

And I'm aware of the Crowdin downsides - my main problem is: Weblate has an UI, non-programers can use, if it got better I'll migrate all projects to it ...

tldr: sadly if you try to convince e.g. your sister to translate stuff it wont work with weblate well :(

**Comment by comradekingu** [comment]
@6543 If you mean keep drive-by translators out, it isn't much of an issue with the level of overview offered. Try the Zen mode if you want something more functional.
My problem is I can't do good work on Crowdin, and even if I am alone there, it is a lot more work than it needs to be.

**Comment by 6543** [comment]
ok I'll exclude Norwegian for crowdin - and what do you rever to by "Zen mode" ?

**Comment by comradekingu** [comment]
@6543 Zen mode in Weblate is the one I use. Similar to Pootle if you used that.
https://docs.weblate.org/en/latest/user/translating.html?highlight=zen#zen-mode

**Comment by helloworld1** [comment]
@comradekingu Is the PR ready to merge?

**Comment by comradekingu** [comment]
@helloworld1 I'd rather sink the effort.
For the month in passing, the source strings aren't fixed, no strings have changed on Crowdin, and I made
https://github.com/andOTP/andOTP/pull/942/files

**Comment by helloworld1** [comment]
Closing for no additional actions.
